docs/docs/sdks/typescript/index.md

```diff
@@ -81,7 +81,7 @@ You may also need to import some definitions from the SDK library:
 ```typescript
 import {
   Identity, ConnectionId, Event, ReducerEvent
-} from '@clockworklabs/spacetimedb-sdk';
+} from 'spacetimedb';
```

Comment:
can you add a note somewhere in this page that SpacetimeDB 1.4.0+ CLI is required for this to work properly?

Reply:
Done